### PR TITLE
fix: fixed profile layout

### PR DESCRIPTION
**Pull request by GaelCondeLosada**
# What I did
Fixed the broken profile layout.

# How I did it
Modified existing style and profile files.

# How to verify it
Test it on expo go.

# Demo 

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [x] work correctly on Android
- [x] are fully tested

### CODE REVIEW

File: screens/Profile/MyProfileScreen/MyProfileScreen.tsx [hunk 1]
```diff
@@ -45,12 +43,10 @@ export const MyProfileScreen = ({ navigation }: MyProfileScreenProps) => {
   }
   return (
     <View style={styles.container}>
-      <View style={[profileStyles.profileContainer, {paddingTop: insets.top + 5}]}>
-        <View style={styles.header}>
-          <TouchableOpacity onPress={() => navigation.goBack()} testID="back-button">
-            <Ionicons name="arrow-back-outline" size={24} color={peach} />
-          </TouchableOpacity>
-        </View>
```
Comment (Aidasvenc) [inline screens/Profile/MyProfileScreen/MyProfileScreen.tsx:53]: This should not have been done, we still need a back button :(

**Review by Aidasvenc — CHANGES_REQUESTED** [review]
We need the back button back. Otherwise, looks good!

**Comment by GaelCondeLosada** [comment]
> We need the back button back. Otherwise, looks good!

You're right, it should be all good now!

**Review by Aidasvenc — APPROVED** [review]
LGTM now!

**Pull request merged**